package/yast2-bootloader.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Jun 14 14:54:38 UTC 2017 - jreidinger@suse.com
+
+- Use udev device for prep partition if it is available
```

Comment:
No. There are no udev devices. There are only udev device **names**.
Also, which is the thing that is available: the udev name or the prep partition?